packages/frontend/core/src/components/page-list/operation-cell.tsx

```diff
@@ -492,7 +492,13 @@ export const TagOperationCell = ({
       <IconButton
         tooltip={t['Rename']()}
         tooltipOptions={tooltipSideTop}
-        onClick={() => setOpen(true)}
+        onClick={useCallback(
+          (e: React.MouseEvent<HTMLButtonElement>) => {
+            e.preventDefault();
```

Comment:
I mean this

Reply:
I am sorry?

Reply:
this is preventDefault, not stoppropagation